purchase_order_type_product_categ/views/purchase_order_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="purchase_order_form" model="ir.ui.view">
```

Comment:
Add `tree` view with this field as optional, hidden by default.

Reply:
I'm sorry, I meant adding it to _purchase order type_ `tree` view, not purchase order. Anyway, right definition is `optional="hide"`, not hidden (I was writing in a human way, was not pseudo code).